### PR TITLE
docs: Add how to verify the binaries are FIPS-compliant

### PR DESCRIPTION
**Pull request by lsgunn-teleport**
Add a link to installation instructions instead of signing into your Teleport account.
Add instructions for download and install from the tarball (instead of partial instructions).
Add how to verify the binaries are from the FIPS package using the `version` command.

Related issue: https://github.com/gravitational/teleport/issues/17966

### CODE REVIEW

File: docs/pages/access-controls/compliance-frameworks/fedramp.mdx [hunk 1]
```diff
@@ -25,29 +27,48 @@ government agencies.
...
+For example, you can download and install from the compressed archive by running the following commands:
 
 ```code
-$ tar -xzf teleport-ent-v(=teleport.version=)-linux-amd64-fips-bin.tar.gz
+$ curl https://get.gravitational.com/teleport-ent-(= teleport.version =)-linux-<Var name="$SYSTEM_ARCH"/>-fips-bin.tar.gz.sha256
+<checksum> <filename>
+$ curl -O https://cdn.teleport.dev/teleport-ent-(= teleport.version =)-linux-<Var name="$SYSTEM_ARCH"/>-fips-bin.tar.gz
+$ shasum -a 256 teleport-ent-(= teleport.version =)-linux-<Var name="$SYSTEM_ARCH"/>-fips-bin.tar.gz
+
+Verify that the checksums match
```
Comment (zmb3) [inline docs/pages/access-controls/compliance-frameworks/fedramp.mdx:44]:
```suggestion
# Verify that the checksums match
$ shasum -a 256 teleport-ent-(= teleport.version =)-linux-<Var name="$SYSTEM_ARCH"/>-fips-bin.tar.gz

```

File: docs/pages/access-controls/compliance-frameworks/fedramp.mdx [hunk 2]
```diff
@@ -25,29 +27,48 @@ government agencies.
...
+$ tar -xvf teleport-ent-(= teleport.version =)-linux-<Var name="$SYSTEM_ARCH"/>-fips-bin.tar.gz
 $ cd teleport-ent
 $ sudo ./install
-# This will copy Teleport Enterprise to /usr/local/bin.
 ```
 
-## Configuration
+After you download and install, all of the Teleport Enterrpise binaries are 
+installed in the `/usr/local/bin` directory. You can verify you have FIPS-compliant
+binaries installed by running the `version` command and verifying that the `X:boringcrypto`
```
Comment (zmb3) [inline docs/pages/access-controls/compliance-frameworks/fedramp.mdx:52]:
```suggestion
binaries installed by running the `teleport version` command and verifying that the `X:boringcrypto`
```

**Review by ptgott — APPROVED** [review]

File: docs/pages/access-controls/compliance-frameworks/fedramp.mdx [hunk 3]
```diff
@@ -25,29 +27,43 @@ government agencies.
...
+installed in the `/usr/local/bin` directory. You can verify you have FIPS-compliant
+binaries installed by running the `teleport version` command and verifying that 
+the `X:boringcrypto` library is listed. For example:
+
+```code
+$ teleport version
+Teleport Enterprise v13.3.9 git:api/v13.3.9-0-gd1a97d9 go1.20.8 X:boringcrypto
+```
+
+If your Teleport cluster runs on AWS, the cluster must run in a GovCloud
```
Comment (reedloden) [inline docs/pages/access-controls/compliance-frameworks/fedramp.mdx:61]: This isn't correct. AWS US East/West can be used for FedRAMP authorized services up to Moderate impact level.

File: docs/pages/access-controls/compliance-frameworks/fedramp.mdx [hunk 4]
```diff
@@ -25,29 +27,44 @@ government agencies.
...
 
-## Configuration
+After you download and install, all of the Teleport Enterprise binaries are 
+installed in the `/usr/local/bin` directory. You can verify you have FIPS-compliant
+binaries installed by running the `teleport version` command and verifying that 
+the `X:boringcrypto` library is listed. For example:
+
+```code
+$ teleport version
+Teleport Enterprise v13.3.9 git:api/v13.3.9-0-gd1a97d9 go1.20.8 X:boringcrypto
```
Comment (stevenGravy) [inline docs/pages/access-controls/compliance-frameworks/fedramp.mdx:58]: Use variables for version and go, not hardcoded.

**Review by stevenGravy — APPROVED** [review]
Approving, please update the variables as commented.

**Comment by ptgott** [comment]
@lsgunn-teleport Is there a manual backport for this?

**Comment by lsgunn-teleport** [comment]
@ptgott not yet. I'm kinda stumped on how to get a clean branch with just my changes in it. I tried to do a cherry-pick of the commits in order, but when I hit this commit, it's a fatal error:
[Links to headings changes](https://github.com/gravitational/teleport/pull/32169/commits/9cef1b326c83c9a8af570ff631d2c41eaeffad69)

I thought I might be able to do something like
`git rebase -i HEAD~9` but that doesn't give me all of my commits to the branch. I still only see the first three.

**Pull request merged**